run/GCClassic/runScriptSamples/operational_examples/nasa_discover/gcc.discover.run

```diff
@@ -1,11 +1,13 @@
 #!/bin/bash
 
 # For discover
-#SBATCH --time 00:45:00
-#SBATCH --nodes=1 --ntasks-per-node=24
-#SBATCH --job-name=gchp_build
-#SBATCH --constraint=sky
-#SBATCH --account=s____
+#SBATCH --time 1:00:00
+#SBATCH --nodes=1 --ntasks-per-node=48
+#SBATCH --mem 4G         # Use 10G for 2x2.5
+#SBATCH --job-name=gcclassic.run
+#SBATCH --constraint=mil # Edit as needed
+#SBATCH --account=s___   # Fill in
+##SBATCH --qos=__        # Fill in and uncomment if using
 
-source gcc.discover.env
+source gcc.env  # Edit, or create gcc.env symlink to env file
 ./gcclassic > gcc.log
```

Comment:
Would recommend doing 
```bash
./gcclassic > gcc.log 2>&1
```
to combine stderr into stdout so that the error messages are in the log file.  (Unless there is a reason that this isn't done on discover)

Reply:
I'll leave this as is for now until we change the standard scripts to have that behavior too.